docs/guide/controller-programming.md

```diff
@@ -452,6 +452,47 @@ if (finished) {
 
 Note that the exit status as well as the value returned by the `main` function are ignored by Webots.
 
+### Console Output
+
+As mentioned earlier, printing to `stdout` or `stderr` from a controller will be redirected by default to the Webots console.
+Note however, that the Webots console doesn't support `stdin` input.
+Like most terminals, it supports a few basic [ANSI escape codes](https://en.wikipedia.org/wiki/ANSI_escape_code) for setting text styles and clearing the content of the console:
+  - 3-bit color (foreground and background)
+  - Bold style
+  - Underline style
+  - Clear screen (same as issuing `clear` command in your terminal)
+  - Reset (colors and styles)
+
+There is an example world in webots: samples/howto/console or at `$WEBOTS_HOME/projects/samples/howto/controllers/console` to demonstrate how to use those.
```

Comment:
From here, this part doesn't look good on the documentation, how may we improve it so it won't look like all stuck together ?

https://cyberbotics.com/doc/guide/controller-programming?version=enhancement-rebase-console-ansi-codes#console-output

Reply:
See my comment in the review.